test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectProtocolValidatorTests.cs

```diff
@@ -271,7 +271,7 @@ public void ValidateMessageWithIdTokenCode()
             var cHashClaim = IdentityUtilities.CreateHashClaim(validCode, "SHA256");
             var jwt = CreateValidatedIdToken();
             jwt.Payload.AddClaim(new Claim(JwtRegisteredClaimNames.Nonce, validNonce));
-            jwt.Header[JwtHeaderParameterNames.Alg] = "SHA256";
+            jwt.Header[JwtHeaderParameterNames.Alg] = JwtAlgorithms.RSA_SHA256;
```

Comment:
Same remark as above, SHA256 is not a signature algorithm and is not a valid valid for `alg`.